lib/src/rollbar_request.dart

```diff
@@ -7,21 +7,20 @@ class RollbarRequest {
 
   RollbarRequest(this._accessToken, this._data, this._logger);
 
-  Future<HttpRequest> send() {
+  Future<Response> send() {
     var json = JSON.encode({"access_token": _accessToken, "data": _data});
 
-    var request = HttpRequest.request("https://api.rollbar.com/api/1/item/",
-        method: "POST",
-        requestHeaders: {"Content-Type": "application/json"},
-        sendData: json);
+    var request = post("https://api.rollbar.com/api/1/item/",
```

Comment:
Did you check that this works in the browser? Looking at the `http` source code, it uses `IOClient` internally.